spec/unit/utils/changelog_reader_spec.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2008-2017, Puzzle ITC GmbH. This file is part of
+#  Cryptopus and licensed under the Affero General Public License version 3 or later.
+#  See the COPYING file at the top-level directory or at
+#  https://github.com/puzzle/cryptopus.
+
+require 'rails_helper'
+describe ChangelogReader do
+
+  context 'verify CHANGELOG.md' do
+    it 'verifies every changelog entry has a version' do
+      ChangelogReader.changelog.each do |changelog_entry|
+        version = changelog_entry.version
+        expect(version).to_not be_nil
+      end
+    end
+
+    it 'verifies every changlog entry has at least one entry' do
+      ChangelogReader.changelog.each do |changelog_entry|
+        expect(changelog_entry.log_entries).to_not be_nil
+      end
+    end
+  end
+
+  context 'test ChangelogReader' do
+
+    subject { ChangelogReader.new }
```

Comment:
check right usage of subject in rspec ... 
maybe it's better to just create let(:changelog_reader) .... in this case